src/reducers/entities/groups.js

```diff
@@ -69,6 +69,9 @@ function syncables(state = {}, action) {
         };
     }
     case GroupTypes.UNLINKED_GROUP_TEAM: {
+        if (!state[action.data.group_id]) {
+            return state;
+        }
```

Comment:
Added this check because now the `UNLINKED_GROUP_TEAM` reducer is being invoked without the `syncables` state being populated first.